neurobooth_terra/redcap.py

```diff
@@ -158,14 +158,38 @@ def extract_field_annotation(s):
     # in redcap speak have a syntax and cannot have other
     # annotations following them separated by space.
     # Those have to go in the next line.
-    # This replace handles those cases
+
+    # Previously we used to parse field annotations by splitting
+    # on space. As we now use complex action equations, it is no
+    # longer possible to split field annotations by space.
+    # Therefore going forward we separate field annotations by a
+    # pipe character '|' and use it to split and parse the
+    # annotations
```

Comment:
what happens if there is a pipe in the content?

Reply:
Field annotations are one of two things - formulas with a defined syntax, or text that is manually added by REDCap users, in our case RCs. While we do not expect anyone to add a pipe character as part of a text annotation, the RCs have added this clause in the REDCap SOP document to use pipe only as a separation character.